style.css

```diff
@@ -0,0 +1,64 @@
+body {
+    background-color: #efece7;
+}
+header,
+section,
+footer {
+	margin: 0 auto;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Magazine_page1/pullRequest?prid=5459324)